_docs/style/spacing.md

```diff
@@ -0,0 +1,290 @@
+      <td class="align-center">&times;</td>
+      <td class="align-center">1.5</td>
+      <td>=</td>
+      <td>{{style.lineHeight}}</td>
+    </tr>{% endfor %}
+  </tbody>
+</table>
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+</section>
+
+<section class="static-section" markdown="1">
+### Headers 
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+
+All content is stacked vertically, yet using line-height values alone does not convey enough visual hierarchy and can result in overly dense layouts. 
+
+Priyanka’s framework provides a solution for this using header and leaf-node margins. Following her example, we experimented with different spacing values, in several layouts, and found that a single spacing value of eight pixels worked best for all header margins. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption: "Header margin (magenta) is stacked below content box (blue)."][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-header-margins.png){: width="492px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+</section>
+
+<section class="static-section" markdown="1">
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-12" %}
+
+### Leaf-nodes 
+
+Next we defined spacing values for leaf-node content. Unlike header margins, it is impossible to use a single spacing value across all types of leaf-node content. For each type of leaf-node content, the spacing is as follows:
+
+* [Table cells](#table-cells): 8 pixels
+* [List items](#list-items): 4 pixels 
+* [Form elements](#form-elements): 4, 8, or 16 pixels, depending on the element
+* [Paragraphs](#paragraphs): 16 pixels
+
+{% endcolumn %}
+
+</div>
+
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### Table cells
+
+Tables are used when data is much more dense and has many attributes. Therefore, if the table’s data is spaced too close, it can be difficult to read through an entire row, without getting distracted by adjacent row data. 
+
+By prescribing eight pixels between table text and table row borders, we get a total of sixteen pixels of space between text across two rows. 
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption: "Table leaf-node spacing"][class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/style/spacing/spacing-system-leaf-node-margins.png){: width="566px"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div class="hxRow"  markdown="1">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+#### List items
+
+A list is a structure made of multiple items of homogenous data. It is important that the list items are not spaced apart like paragraphs, which typically hold heterogenous ideas.
```

Comment:
Cross link to lists.html; Also we will want to update the images to detail this new 4px space between if this was not done prior.

Reply:
yep -- will need to update the lists article with spacing specs.  I might do that real quick...